### ISSUE TITLE
[A11y][ScreenReader]NVDA is announcing extra information when the user activates the shortcut key for the heading 'H'.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9173]
Original: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1555443

Repro Steps:
Hit https://www.nuget.orgb/ and press Enter key.
Navigate to "Sign in" control and hit enter.
Navigate to "Sign in with Microsoft" control and hit enter.
Search for a key term (say “json”) from search box.
Click on "Newtonsoft.json" control and hit enter.
Navigate to "Info" and "Statistics" section present on right of the screen.
Verify that the screen reader is announcing correctly or not when the heading shortcut key is activated on the Newtonsoft.json heading.

Note: 
This issue repros with NVDA only.

Actual Result:
NVDA is announcing a lot of extra information when the user activates the shortcut key for the heading 'H'.

Expected result:
NVDA should be announce the heading of Newtonsoft.json heading and its level when the user activated the heading shortcut key.

User Impact:
Screen Reader Users may be get confused when the screen reader should announce the whole thing in one go when the user activated the heading shortcut key.

Notes:
I am unsure what NVDA considers to be the heading, but it is likely there is too much stuff under what NVDA considers the heading (especially considering that the issue does not repro on Narrator). Any fix we try here would need to be verified in Narrator so we can make sure we don't break that scenario.

**Comment by camigthompson** [comment]
Verified in PROD.
